pkgs/top-level/all-packages.nix

```diff
@@ -18352,6 +18352,8 @@ with pkgs;
     erlang = erlang_25;
   };
 
+  m5ops = callPackage ../by-name/ge/gem5/m5ops.nix { };
```

Comment:
I don't think this is necessary anymore.

Reply:
Just tried removing it and I get a missing attribute error. I think it's because I'm using a non-standard name within the gem5 parent directory. I put the package in there cause it's very closely related and should pretty much always be used with gem5 even though the builds are separate.

Should I put it in its own directory or is it fine to leave this?

Reply:
Yeah, you're right, it needs to be package.nix. To be honest, I'm not sure. It probably needs its own directory, but someone more knowledgeable than me should answer that.

Reply:
:+1: ok. If you search for `by-name.*\{ \}` in `all-packages.nix` there are a couple other examples of this, so I'll leave it for now, but if someone has more information about the correct layout lmk.